src/coreclr/src/gc/gcinterface.h

```diff
@@ -782,7 +779,7 @@ class IGCHeap {
     virtual Object* AllocAlign8(gc_alloc_context* acontext, size_t size, uint32_t flags) = 0;
 
     // This is for the allocator to indicate it's done allocating a large object during a
-    // background GC as the BGC threads also need to walk LOH.
+    // background GC as the BGC threads also need to walk UOH.
```

Comment:
What does UOH stand for?

Reply:
User's Old Heaps or User-allocated Other Heaps.

The unifying property is that they are not SOH - i.e. they do not participate in generational behavior. Nothing will promote into or out of these.
That implies they are logically in the oldest generation. As long as card tables are one-directional, these will have to be marked together with Gen2.

It was hard to name ;-)

Reply:
It is weird name. It implies that there are non-user heaps or non-user allocated heaps. Do we have such non-user heaps?

> they are logically in the oldest generation

This would lead to a name like "Oldest Generation Heap", OGH.

Reply:
It is mostly an internal term since the concept of "other than SOH" must have a name. We use it in many places - like comments enums, function and type suffixes. 
If there is a definitely better one, we can change to that (not in this PR).

Ideally the new name would abbreviate as UOH, of course - then no need to change anything. :-)

Reply:
We do use these internal terms to explain how the GC works to our customers. So it is somewhat important that it is a good name.

Reply:
I logged an issue to follow up on this - https://github.com/dotnet/runtime/issues/33445